index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
Gonna keep this as a TODO, I think the real problem here is that normally testem will start the browser pointed to the server and port. But since we're serving off of a file we don't really have that option, so the client needs to find its port. I think its a bit less straight forward than I thought...